src/System.Data.Common/src/System/Data/ProviderBase/SchemaMapping.cs

```diff
@@ -329,16 +323,11 @@ private object[] GetMappedValues()
                             }
                             else
                             {
-                                switch (_xmlMap[i])
+                                _readerDataValues[i] = _xmlMap[i] switch
                                 {
-                                    case SqlXml:
-                                        // map strongly typed SqlString.Null to SqlXml.Null
-                                        _readerDataValues[i] = System.Data.SqlTypes.SqlXml.Null;
-                                        break;
-                                    default:
-                                        _readerDataValues[i] = DBNull.Value;
-                                        break;
-                                }
+                                    SqlXml => System.Data.SqlTypes.SqlXml.Null, // map strongly typed SqlString.Null to SqlXml.Null
```

Comment:
For many of these I'd invert the comment... like this:

```suggestion
                                    // map strongly typed SqlString.Null to 
                                    SqlXml => System.Data.SqlTypes.SqlXml.Null,SqlXml.Null,
```

(and with a blank line before it if the previous line wasn't the `{`).

But that's partly because I dislike trailing comments, and partly because gramattically speaking, the comment is after the comma, so it's with the next thing.

Reply:
> partly because gramattically speaking, the comment is after the comma, so it's with the next thing.

I think that's the same thing as disliking trailing comments.  In most situations with a comment at the end of the line, it's after punctuation, e.g. after a ';' demarcating the end of a statement, which "grammatically speaking" would make it part of the next thing.  Personally, I don't think such English-grammar-based-reasoning really applies to comments like this.